quickstarts/microsoft.kusto/kusto-event-hub/main.bicep

```diff
@@ -0,0 +1,111 @@
+@description('Location for all resources')
+param location string = resourceGroup().location
+@description('Name of the cluster')
+param clusterName string = 'kusto${uniqueString(resourceGroup().id)}'
+@description('Name of the database')
+param databaseName string = 'kustodb'
+@description('Name of Event Hub\'s namespace')
+param eventHubNamespaceName string = 'eventHub${uniqueString(resourceGroup().id)}'
```

Comment:
are eventhub names globally unique?

Reply:
Yes as they map to a domain name (maybe region-global)